jupyter/utils/install_tex_live.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash 
+echo "Installing TexLive to allow PDf export from Notebooks" 
+curl -L https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz -o install-tl-unx.tar.gz 
+zcat < install-tl-unx.tar.gz | tar xf -
+cd install-tl-2*
+perl ./install-tl --no-interaction --scheme=scheme-small
+cd /usr/local/texlive/2025/bin/x86_64-linux
```

Comment:
Shall we not include `2025` term in here, 
keep this future scoped, or else we would need to keep modifying this
WDYT ?